.github/ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,16 @@
 <!--
-Thanks for wanting to report an issue you've found in Node.js. Please fill in
-the template below by replacing the html comments with an appropriate answer.
-If unsure about something, just do as best as you're able.
+Thank you for reporting an issue. Please fill in the template below. If unsure
+about something, just do as best as you're able.
 
 version: usually output of `node -v`
```

Comment:
What about capitalizing the field names here to match the actual content (and to make them stand out better)?